web/tests/pageobjects/page.ts

```diff
@@ -55,6 +56,79 @@ export default class Page {
         await browser.keys(Key.Tab);
     }
 
+    async setSearchSelect(name: string, value: string) {
```

Comment:
This can probably be deleted soon.  Although I have the thought that it might make sense to move all of the `controls` into a JS Object so that they can all have a _local_ context object; that way, the `$` WebdriverIO context can be localized to, say, _this form_ or _this wizard panel_, and we wouldn't have to struggle with "stale" (really, fragmented-by-Lit) objects on the page sharing fieldname identifiers.  That's a thought for the future, though.